### ISSUE TITLE
Crashing in the Nether - java.lang.IllegalStateException: Server tried to update attributes of a non-living entity

### ISSUE THREAD

**Issue by Cheremenin** [SleepyTrousers/EnderIO#4471]
When going into the Nether on my server some "weather rocket" entity seems to spawn after about 10 seconds and crashes me. After removing and recreating the Nether the same thing occurs. Tried copying my server's config folder to my client to "sync" them since it's apparently the cause, still crashes.

- EnderIO: 1.7.10-2.3.0.429_beta.jar
- EnderCore: 1.7.10-0.2.0.39_beta.jar
- Minecraft: 1.7.10
- Forge: 10.13.4.1614

Any ideas?

Here's the log:
[crash-2018-03-07_15.41.03-client.txt](https://github.com/SleepyTrousers/EnderIO/files/1789014/crash-2018-03-07_15.41.03-client.txt)

**Comment by mindforger** [comment]
tl;dr; your EIO configs are out of sync, resync them, this may be caused by an mismatch in mob IDs

i did remember this one but i was usure, it is most likely the witherskelleton being a separat mob id for EIO to spawn them as powered spawner in the overworld, and either your server or your client has witherskelleton separation disabled

i recommend to use the server file for your client config, this way you won't have to go through an worldupdate on the server

**Comment by HenryLoenwind** [comment]
I suspect there's some mod out there that messes up entity IDs, especially when the Wither Skeleton replacement is enabled. We never for the reason for this or the Wither Skeleton "pileup" and both certainly don't happen with just Ender IO.

**Comment by Cheremenin** [comment]
B:replaceWitherSkeletons=false on the server, no longer crashing. Likely was the cause, not sure though.

As far as I understand it was creating a whole different new entity while spawning wither skeletons which was conflicting with something else (that weather rocket from the same mod maybe?) and crashing me?

**Comment by mindforger** [comment]
>As far as I understand it was creating a whole different new entity while spawning wither skeletons which was conflicting with something else (that weather rocket from the same mod maybe?) and crashing me?

yes

to be more precise, the game spawned an entity with a mob id, transfered this ifnormations to your client and your client has another another mob associated with that id and while it tried to access informations to render the entity it crashed

**Comment by HenryLoenwind** [comment]
Some other mod conflicting with it somehow and shifting entity IDs because of it. That it hit the weather rocket doesn't have anything to do with the rocket, it's just that the rocket has the ID after our Wither Skeleton. Or, if there is no EIO Skeleton, it has the ID that one would have.
